app/models/recipe.rb

```diff
@@ -5,6 +5,7 @@ class Recipe < ApplicationRecord
   has_many :sporks, foreign_key: "original_recipe_id"
 
   validates :title, presence: true
+  validates_length_of :title, :maximum => 30
```

Comment:
The ```:maximum => 30``` is the older syntax for a hash key symbol keys. Now you can save some typing by using the newer hash symbol:key shorthand:
```validates_length_of :title, maximum: 30```
which is actually doing this:
```validates_length_of(:title, { :maximum => 30 })```